Update the Org Repository Policy per the 2022 PSC (#1036)
python/devguide: developer-workflow/development-cycle.rst

* Helpers and backports for Python/CPython features (e.g. `typing_extensions <https://github.com/python/typing_extensions>`_, `typeshed <https://github.com/python/typeshed>`_, `tzdata <https://github.com/python/tzdata>`_, `pythoncapi-compat <https://github.com/python/pythoncapi-compat>`_)
* Organization-related repositories (e.g. the `Code of Conduct <https://github.com/python/pycon-code-of-conduct>`_, `.github <https://github.com/python/.github>`_)
* Documentation and websites for all the above (e.g. `python.org repository <https://github.com/python/pythondotorg>`_, `PEPs <https://github.com/python/peps>`_, `Devguide <https://github.com/python/devguide>`_, docs translations)
* Infrastructure for all the above (e.g. `docsbuild-scripts <https://github.com/python/docsbuild-scripts>`_, `buildmaster-config <https://github.com/python/buildmaster-config>`_)
* Discussions and notes around official development-related processes and events (e.g. `steering-council <https://github.com/python/steering-council>`_, `core-sprint <https://github.com/python/core-sprint>`_)

> Before adding a new repository to the organization, have discussion to seeking
> consensus in our `Committers forum <https://discuss.python.org/c/committers/5>`_.
> Once people are satisfied with that, ask the `Python steering council <https://github.com/python/steering-council>`_
> to grant permission.

Reviewer: Both of these are ungrammatical for multiple reasons; here's an alternative that is (there are number of other options, too):

```suggestion
Before adding a new repository to the organization, open a discussion to seek consensus
in the `Committers Discourse category <https://discuss.python.org/c/committers/5>`_.
Once people are satisfied with that, ask the `Python steering council <https://github.com/python/steering-council>`_
to grant permission.
```

Note that several repositories remain in the organization for historic reasons,
and would probably not be appropriate to add today.

Generally, new repositories should start their life under personal GitHub
accounts or other GitHub orgs. It is relatively easy to move a repository to
the organization once it is mature. For example, this would now apply to
experimental features like `asyncio <https://github.com/python/asyncio>`_,
`exceptiongroups <https://github.com/python/exceptiongroups>`_, or

> `typed_ast <https://github.com/python/typed_ast>`_, and drafts of new guides

Reviewer 2: I'm not sure this is quite historically accurate. `typed-ast` was originally created to support typecheckers, similar to typeshed, without intention to move it to CPython.

Author: thanks, we're listing more than is needed to make the point. I just took typed_ast out of the list.

and other documentation (e.g. `redistributor-guide <https://github.com/python/redistributor-guide>`_).

General-use tools and libraries (e.g. `mypy <https://github.com/python/mypy>`_

> or `black <https://github.com/psf/black>`_) should also be developed outside
> the python organization, unless core devs (as represented by the SC)

Reviewer 3: ```suggestion
or `Black <https://github.com/psf/black>`_) should also be developed outside
the Python organization, unless core devs (as represented by the SC)
```

Reviewer: or

```suggestion
or `black <https://github.com/psf/black>`_) should also be developed outside
the ``python`` organization, unless core devs (as represented by the SC)
```
